src/en/experts-sla.md

```diff
@@ -1,149 +1,257 @@
+Total                   2.42         350               1%   
+Wallet                              1000                    
+Unallocated                          650
 ```
 
-The budget limit may be lowered, but not below the current spend.
+The budget limit may be lowered for a model, but not below its current spent
+value.
 
-## Advanced budgeting
+### Advanced budgeting
 
-You may create additional wallets for accounting or organizational purposes.
+Additional wallets may be created for accounting or organizational purposes
+with the `create-wallet` command:
 
 ```bash
-juju create-wallet demo-poc 200
-wallet created
+juju create-wallet demo 50
 ```
 
+Verify the creation of the new wallet by listing the wallets again:
+
 ```bash
 juju wallets
+```
+
+Sample output:
+
+```no-highlight
 Wallet          Monthly Budgeted        Available       Spent
-demo-poc            200        0           200.00        0.00
-personal*          1000      100          1000.00        0.00
-Total              1200      100          1200.00        0.00
+demo                 50        0            50.00        0.00
+primary*           1000      350           997.58        2.42
+Total              1050      350          1047.58        2.42
                                                              
-Credit limit:     10000                                      
+Credit limit:     10000
 ```
 
-To allocate a budget from a specific wallet, use _wallet_:_limit_ when setting
-the budget on an SLA.
+To allocate a budget (using either the `sla` command or the `budget` command)
+from a specific wallet use the form: `<wallet>:<limit>`. Here are two
+examples:
 
 ```bash
-juju sla essential demo-poc:50
+juju sla essential demo:40
+juju budget primary:25
 ```
+
+## Metering
+
+Metering begins once the following have occurred:
+
+ - an SLA has been assigned to a model
+ - a budget has been allocated to a model
+ - the model takes on a workload
+ 
+When metering is underway the output to the `status` command will include an
+extra section:
+
+```no-highlight
+.
+.
+Entity  Meter status  Message
+model   green
+.
+.
+```
+
+A 'Meter' value of **green** (displayed with a green colour in the terminal)
+indicates that the customer’s payment method is in good standing. A value of
+**amber** indicates a condition puts future payment for continued use at risk
+(e.g. spending is occurring at a rate likely to exceed the monthly limit). A
+value of **red** indicates that continued use of the workload will not be paid
+for (e.g. budget limit exceeded, problem with payment method). The 'Message'
+column will be updated with appropriate details.
+
+Determining factors for actual metering costs (e.g. fixed per-hour rate,
+per-unit hour, etc.) are negotiated between you and your Expert.
+
+## Creating a support case
+
+When it comes time to request help from your Expert you can file a support case
+using the same URL for requesting a credit limit increase:
```

Comment:
I'd not tie the url to the credit limit. Just show it as the correct url for someone looking to file an issue.

Reply:
OK, but I thought it would be simpler on the user to know that there is just one URL and not a bunch to keep track of.

Reply:
Right, but if they come looking for a URL for X they're looking for X and anything not X is just noise to have to filter through.

Reply:
It's not a big issue for me. I'll remove the extra words.